transactions/testMintDandyTO.cdc

```diff
@@ -130,7 +130,7 @@ transaction(name: String, maxEdition:UInt64, artist:String, nftName:String, nftD
 			let editioned= MetadataViews.Edition(name: "nft", number:i, max:maxEdition)
 			let artCreativeWork=FindViews.CreativeWork(artist: artist, name: "flovatar ", description: "flovatar_NFT", type:"image")
 			let description=artCreativeWork.description.concat( " edition ").concat(i.toString()).concat( " of ").concat(maxEdition.toString())
-			let artHttpFile=MetadataViews.HTTPFile(url:"https://flovatar.com/api/image/166")
+			let artHttpFile=MetadataViews.HTTPFile(url:"https://flovatar.com/api/image/166 ")
```

Comment:
ths whitespace here is wrong

Reply:
the id 166 is a little embarrasing because it will get replaced by "replaceID" function.
I will just change it to other ones to avoid that.